### ISSUE TITLE
No Steam 

### ISSUE THREAD

**Issue by Mantasesurvives** [BizakDaTroll/WindowsGSM.Icarus#1]
-nosteam shows up in the server name, it is pulling from the code in the CS, I tried to remove it and just use -log but the server wouldn't start. 
So when I add -SteamServerName"survival gaming club"  on the server list it displays

Survival Gaming Club -nosteam 

Everything works fine, it is just ugly looking and not accurate. 
Thanks

**Comment by BizakDaTroll** [comment]
Yeah its a quoting issue I believe. Need to find some time to fix

**Comment by Mantasesurvives** [comment]
Im guessing you saw the second issue, I already fixed the code you just need to copy paste. I couldn't figure out how to do it. 

Let me restate it, I changed it in your file and ran my server , it was fixed, What i couldn't figure out was how to upload it to github as a program.

**Comment by BizakDaTroll** [comment]
Should be fixed
